sql-db/sql-app-oracle/src/test/java/io/quarkus/ts/sqldb/sqlapp/oracle/AbstractSqlDatabaseIT.java

```diff
@@ -0,0 +1,197 @@
+                .body("code", equalTo(HttpStatus.SC_UNPROCESSABLE_ENTITY))
+                .body("error", equalTo("unexpected ID in request"));
+    }
+
+    @Test
+    @Order(6)
+    public void update() {
+        Book book = new Book();
+        book.id = Long.valueOf(VALID_ID);
+        book.title = "Schismatrix";
+        book.author = "Bruce Sterling";
+
+        given()
+                .contentType(ContentType.JSON)
+                .body(book)
+                .put("/book/8")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .body("id", equalTo(VALID_ID))
+                .body("title", equalTo("Schismatrix"))
+                .body("author", equalTo("Bruce Sterling"));
+
+        given()
+                .get("/book/" + VALID_ID)
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .body("title", equalTo("Schismatrix"))
+                .body("author", equalTo("Bruce Sterling"));
+    }
+
+    @Test
+    @Order(7)
+    public void updateWithUnknownId() {
+        Book book = new Book();
+        book.id = Long.valueOf(INVALID_ID);
+        book.title = "foo";
+        book.author = "bar";
+
+        given()
+                .contentType(ContentType.JSON)
+                .body(book)
+                .put("/book/999")
+                .then()
+                .statusCode(HttpStatus.SC_NOT_FOUND)
+                .body("code", equalTo(HttpStatus.SC_NOT_FOUND))
+                .body("error", equalTo("book '999' not found"));
+    }
+
+    @Test
+    @Order(8)
+    public void updateInvalidPayload() {
+        given()
+                .contentType(ContentType.TEXT)
+                .body("")
+                .put("/book/8")
+                .then()
+                .statusCode(HttpStatus.SC_UNSUPPORTED_MEDIA_TYPE)
+                .body("code", equalTo(HttpStatus.SC_UNSUPPORTED_MEDIA_TYPE));
+    }
+
+    @Test
+    @Order(9)
+    public void updateBadPayload() {
+        Book book = new Book();
+
+        given()
+                .contentType(ContentType.JSON)
+                .body(book)
+                .put("/book/8")
+                .then()
+                .statusCode(HttpStatus.SC_UNPROCESSABLE_ENTITY)
+                .body("code", equalTo(HttpStatus.SC_UNPROCESSABLE_ENTITY))
+                .body("error.message", containsInAnyOrder("book title must be set", "book author must be set"));
+    }
+
+    @Test
+    @Order(10)
+    public void delete() {
+        given()
+                .delete("/book/8")
+                .then()
+                .statusCode(HttpStatus.SC_NO_CONTENT);
+
+        given()
+                .get("/book/8")
+                .then()
+                .statusCode(HttpStatus.SC_NOT_FOUND)
+                .body("code", equalTo(HttpStatus.SC_NOT_FOUND))
+                .body("error", equalTo("book '8' not found"));
+    }
+
+    @Test
+    @Order(11)
+    public void deleteWithUnknownId() {
```

Comment:
If I am not mistaken, this test checks that Book.findById(id) properly returns null. Since it doesn't check any DELETE-specific code, I suggest to use GET method here, for the sake of clarity.

Reply:
You're right, let me replace it with `getWithUnknownId`. I think the same goes for `updateWithUnknownId`.